docs/pouch_with_cri_introduction.md

```diff
@@ -0,0 +1,37 @@
+# pouch with CRI introduction
+
+## what is CRI
+
+The Container Runtime Interface (CRI) is a plugin interface which enables kubelet to use a wide variety of container runtimes, without the need to recompile. 
+CRI (_Container Runtime Interface_) consists of a [protobuf API](https://git.k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto),specifications/requirements and [libraries](https://git.k8s.io/kubernetes/pkg/kubelet/server/streaming)for container runtimes to integrate with kubelet on a node.
+
+This diagram explains how the Container Runtime Interface works:
+
+![CRI works](static_files/pouch_cri_works.png)
+
+## why need to implement CRI in pouch
+
+The purpose of the pouch CRI implementation is to integrate with Kublete.This allow Kubernetes to directly launch and manage containers through the pouch CRI.
+
+## application scene
+
+Manage containers via Kubernetes.
+
+## pouch CRI Create a single-container pod
+
+Let's use an example to demonstrate how pouch-cri works for the case when Kubelet creates a single-container pod:
+
+* Kubelet calls pouch-cri, via the CRI runtime service API, to create a pod.
+* pouch uses containerd to create and start a special pause container (the sandbox container) and put that container inside the pod’s cgroups and namespace (steps omitted for brevity).
```

Comment:
pause -> paused?

Reply:
In Kubernetes, the pause container serves as the "parent container" for all of the containers in pod.